### PR TITLE
Fix some spelling errors

### PR DESCRIPTION
**Pull request by shanselman**
fixing spelling errors

### CODE REVIEW

**Comment by DHowett** [comment]
image

Yay! No spelling errors! 😁

**Comment by shanselman** [comment]
good lord...sorry, was rushing!

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Welcome to the repository for [Sudo for Windows](https://aka.ms/sudo) 🥪. Sudo
 for Windows allows users to run elevated commands direcly from unelevated
```
Comment (asw101) [inline README.md:4]:
```suggestion
for Windows allows users to run elevated commands directly from unelevated

```

File: README.md [hunk 2]
```diff
@@ -1,10 +1,11 @@
 # ![Sudo for Windows icon](./img/Windows/AppList.targetsize-24.png) Sudo for Windows
 
 Welcome to the repository for [Sudo for Windows](https://aka.ms/sudo) 🥪. Sudo
-for Windows allows users to run elevated commands direcly from unelevated
+for Windows allows users to run elevated commands directly from unelevated
+
```
Comment (DHowett) [inline README.md:5]:
```suggestion
```

**Pull request merged**